<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QWidget" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>贪吃蛇</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/test/pic/background.jpg);</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>610</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>500</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Brazil 2014</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome to GLuSnake</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_single">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FZPXJW</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(153, 102, 51, 160)</string>
   </property>
   <property name="text">
    <string>经典单人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_double">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FZPXJW</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(153, 102, 51, 160)
</string>
   </property>
   <property name="text">
    <string>劲爆双人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_AI">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FZPXJW</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(153, 102, 51, 160)
</string>
   </property>
   <property name="text">
    <string>简易AI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_againstAI">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FZPXJW</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(153, 102, 51, 160)
</string>
   </property>
   <property name="text">
    <string>人机大战</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_more">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FZPXJW</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(153, 102, 51, 160)
</string>
   </property>
   <property name="text">
    <string>更多游戏设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
